katello.gemspec

```diff
@@ -57,8 +57,8 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_ansible_client", ">= 0.2", "< 0.7"
   gem.add_dependency "pulp_container_client", ">= 2.0.0", "< 2.3.0"
   gem.add_dependency "pulp_deb_client", ">= 2.6.0", "< 2.9.0"
-  gem.add_dependency "pulp_rpm_client", ">=3.6.2", "< 3.9.0"
-  gem.add_dependency "pulp_2to3_migration_client", ">= 0.7.0", "< 0.8.0"
+  gem.add_dependency "pulp_rpm_client", ">=3.9.0", "< 3.9.1"
```

Comment:
I'm curious about why the pulp_rpm_client version is locked to 3.9.0.  Or did you mean this? 
```ruby
  gem.add_dependency "pulp_rpm_client", ">=3.9.0", "< 3.10.0"
```

Reply:
Actually that is not intentional, and I suppose that should be changed.